packages/block-library/src/post-content/edit.js

```diff
@@ -45,12 +46,29 @@ function EditableContent( { context = {} } ) {
 		{ id: postId }
 	);
 
+	const hasInnerBlocks = useSelect(
+		( select ) =>
+			select( blockEditorStore ).getBlock( clientId ).innerBlocks.length >
+			0,
+		[ clientId ]
+	);
+
+	const initialInnerBlocks = [
+		[
+			'core/paragraph',
+			{
+				placeholder: __( 'Type / to add blocks to your page' ),
```

Comment:
The post content block might not always be in a `page`. E.g. if we one day have a `Posts` screen in browse mode. Would it be worth making this more generic?

Reply:
Good point, might be best to remove this placeholder so the default one appears.